java/statestore/src/main/java/sleeper/statestore/s3/S3FileReferenceStore.java

```diff
@@ -120,6 +121,59 @@ public void addFiles(List<FileReference> fileReferences) throws StateStoreExcept
         }
     }
 
+    @Override
+    public void splitFileReferences(List<SplitFileReferenceRequest> splitRequests) throws StateStoreException {
+        Instant updateTime = clock.instant();
+        for (SplitFileReferenceRequest splitRequest : splitRequests) {
+            splitFileReference(splitRequest, updateTime);
+        }
+    }
+
+    private void splitFileReference(SplitFileReferenceRequest splitRequest, Instant updateTime) throws StateStoreException {
+        FileReference oldReference = splitRequest.getOldReference();
+        String oldPartitionAndFilename = oldReference.getPartitionId() + "|" + oldReference.getFilename();
```

Comment:
Minor, but given we use "|" a lot it would be better to make this a static DELIMITER variable.

Reply:
Done, thanks